pinot-segment-local/src/main/java/org/apache/pinot/segment/local/upsert/UpsertTTLStrategy.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.segment.local.upsert;
+
+import java.util.concurrent.TimeUnit;
+
+
+public class UpsertTTLStrategy {
```

Comment:
can you add javadocs for this class? 

IIuc, this seems to be a class to hold the values from `UpsertTTLConfig` with an additional method to convert to millis. So, is this class really needed?

Reply:
@navina this class is similar to TimeRetentionStrategy in package org.apache.pinot.controller.helix.core.retention.strategy. I added a new class so it can be extended in the future.
E.g. skipLate event etc. Different approach for expired primary keys cleanups, etc.

Reply:
I agree with @navina that this extra class is not really needed, unless we want different TTL handling strategies, at which time an interface should be introduced, instead of a concrete class.

Reply:
@navina @Jackie-Jiang gotcha. Let me merge adding an addition method in upsertTTLConfig and remove this class for now. I can introduce addition interface if needed in future. e.g. two map rotated approach for TTL handling.